sky/clouds/aws.py

```diff
@@ -340,9 +340,6 @@ def get_egress_cost(self, num_gigabytes: float):
         cost += 0.0
         return cost
 
-    def is_same_cloud(self, other: clouds.Cloud):
```

Comment:
Should we also remove `is_same_cloud` from azure.py?

Reply:
Oops, thanks for catching this! Fixed. Thanks!